R/fill_geographies.R

```diff
@@ -126,8 +126,10 @@ fill_postcode_geogs <- function(data) {
 }
 
 fill_gpprac_geographies <- function(data) {
-  gpprac_ref <- read_file(get_slf_gpprac_path()) %>%
-    dplyr::select("gpprac", "cluster", "hbpraccode")
+  gpprac_ref <- read_file(
+    get_slf_gpprac_path(),
+    col_select = c("gpprac", "cluster", "hbpraccode")
+  )
```

Comment:
I've checked other uses for `col_select` in this PR and they are working. For this one, it seems to read in 8 columns? I'm not too sure why as the code looks right to me

Reply:
I think the issue here is that the existing file is an `rds` so it reads that and doesn't use the `col_select` parameter at all. I introduced an error so that if you use the `col_select` or `as_data_frame` parameters on a non `parquet` file it tells you.

I've updated this branch so hopefully you will also see that error... It should all work out once the files have been written out!